app/services/adapters/jwe_webhook_destination.rb

```diff
@@ -0,0 +1,39 @@
+module Adapters
+  class JweWebhookDestination
```

Comment:
Seems strange to have the `jwe` in here? JWE is just the encryption type we are using which might change. Some like just `WebhookDestination` or `SubmissionDestination` explains the use without the implementation

Reply:
I've left `WebhookDestination` intact as that does a POST. I've named it explicitly so someone can do a "normal" webhook call. Also if one were to use `WebhookDestination` they wouldn't expect the body of their call to be randomly encrypted

Reply:
That algorithm explicitly requires the string to be 16 characters in length. Is that an issue? If so I can have a look at what the other options are